client/components/SingleItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import axios from 'axios';
+import { Link } from 'react-router-dom';
+
+export default class SingleItem extends React.Component{
+  constructor() {
+    super();
+    this.state = {
+      singleItem: {
+        reviews: []
+      }
+    }
+  }
+  async componentDidMount(){
+    try {
+      const itemResponse = await axios.get('/api/items/3');
```

Comment:
i know you're probably not finished with this but this also belongs in the redux store